app/shared/accessibility-shortcuts/AccessibilityShortcuts.js

```diff
@@ -0,0 +1,26 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import injectT from '../../i18n/injectT';
+
+function AccessibilityShortcuts({ t, mainContentId }) {
+  const mainContentHref = `#${mainContentId}`;
+
+  return (
+    <div className="app-AccessibilityShortcuts">
+      <a
+        className="sr-only app-AccessibilityShortcuts__skip-link"
+        href={mainContentHref}
```

Comment:
why dont we just `href='main-content'` here and skip all the passing props, it only used once in this case

Reply:
So that we are not managing this ID in two places. If we pass it as a prop, we can define it in a single place. Which in this instance is the `AppWrapper`. Another route could be to have it as a constant, but I thought that this way we could have the "original" ID in the same place as before (`AppWrapper`) which sort of decreases the scope of this change.

Reply:
I just feel its not necessary to pass it around. Cause AppWrapper is the highest level and it actually wrap everything, i can't find any other usage of this

Reply:
Here's one scenario where hard coding the value may be problematic:

If some dev changes the id in `AppWrapper`, and the dev in question is not me (:D), they likely won't know that they are breaking the functionality of the `AccessibilityShortcuts`.

My logic, in other terms, is to avoid similar sort of maintainability issues we try to avoid by using BEM (or CSS modules). By hard coding the values, we end up having implicit relationships. In this instance, because the feature in question will probably be used relatively seldomly by us, it'll be even easier to miss this relationship by accident. And in the case we would happen to miss it, it'd likely take for some time for us to spot it.

If we just hard coded the values, we could add a test that validates that the link skips to the correct place on the page. But I don't think that we currently have test cases on that level--and it'd be a more complicated way to approach this imo (although we should add a test like that if we do for instance e2e at some point).

Now that the ID is defined in a single place, the dev will, by accident, implement their changes in a way that does not break `AccessibilityShortcuts`.

I'd be willing to do this in some other way if you think it's worth it, but I would prefer to avoid having the implicit relationship here.

Reply:
Yes, approved 😄